### ISSUE TITLE
UI not rendering properly when clicked on push notifications in IOS

### ISSUE THREAD

**Issue by ppv94** [facebook/react-native#32621]
### Description

I had integrated push notification in my react native application. when I click on the push notification (removed app instance) my application opens and it navigates to the Home screen but **UI does not render (styles are not applied to UI)**. it happens only in IOS. Android works fine

**my versions**
```
"@aws-amplify/pushnotification": "^3.2.32"
"aws-amplify": "4.3.7"
"aws-amplify-react-native": "6.0.1"
"@react-native-community/push-notification-ios": "1.9.0"
"react": "16.13.1"
"react-native": "0.63.4"
```
I am using _PushNotification.onNotificationOpened_ outside the hooks class.
Styles are loaded from external StyleSheet

```
I am using 
`import PushNotification from '@aws-amplify/pushnotification'`

PushNotification.onNotificationOpened(notification => {
    let obj = notification["_data"]["data"]["jsonBody"]
    setTimeout(()=>{
      notificationAPI(obj.id)
    },1000)
});

const Home = ({route, navigation}) =>{

useEffect(() => {
APICall()
  })
}, [isVisible]);

return (
 <View>
**CUSTOME UI**
</View>
)
}

export default Home;
```

### Version

0.63.4

### Output of `react-native info`

System:
    OS: macOS 10.15.7
    CPU: (8) x64 Intel(R) Core(TM) i7-3615QM CPU @ 2.30GHz
    Memory: 602.55 MB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 14.16.1 - /usr/local/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.14.12 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.10.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    Android SDK:
      API Levels: 23, 28, 29, 30
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.3, 31.0.0
      System Images: android-27 | Google Play Intel x86 Atom, android-29 | Google Play Intel x86 Atom, android-30 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 2020.3 AI-203.7717.56.2031.7621141
    Xcode: 12.4/12D4e - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_222 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.4 => 0.63.4 
    react-native-macos: Not Found

### Steps to reproduce

How can I solve this issue when I click on push notifications UI should render properly with all the styles applied.

Original UI
![25823](https://user-images.githubusercontent.com/28724702/142404883-b42bc5d8-2bc0-4b6b-8207-d7f436650ebb.jpg)

when i click on push notification app opens and styles does not apply.

![97905](https://user-images.githubusercontent.com/28724702/142404525-d86b092b-9ea6-4e67-8bc8-703841b056fc.jpg)

**Comment by ppv94** [comment]
I had updated my Xcode to 13.1 and released build then the issue was resolved.

**Comment by rahult-sdei** [comment]
I had same issue but didnt resolved even after updating the xcode

**Comment by rahult-sdei** [comment]
> Still seeing this issue on xCode 13.3. Is anybody else experiencing this issue?

Please try to use latest firebase notification setup. It will be resolved.
Mine is resolved by using latest firesbase library and setup

**Comment by kukadiyaAni** [comment]
I had same issue but didn't resolved even after updating the xCode to 13.2.1
